sdk/python/foundation-models/system/import/import_model_into_registry.ipynb

```diff
@@ -44,11 +48,10 @@
     "You can still download and register the model using the outputs of the `download_model` job. You need to [write your own inference code](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-deploy-online-endpoints?tabs=python) in this case. It also means that fine tuning is not yet supported if the task type of the model you are interested in is not in the supported list.\n",
     "\n",
     "### Outline\n",
-    "* Setup pre-requisites such as compute.\n",
-    "* Pick a model to import.\n",
-    "* Configure the import job.\n",
-    "* Run the fine tuning job.\n",
-    "* Register the fine tuned model. \n",
+    "1. Connect to Azure Machine Learning Workspace\n",
+    "2. Create a pipeline job using the pipeline component for HuggingFace models\n",
+    "3. Create a pipeline job using components for MMDetection models\n",
```

Comment:
nit: can be 2.1 and 2.2

Reply:
Moved MMD to a separate notebook per Sasi's suggestion offline.